aws/ec2_test.go

```diff
@@ -0,0 +1,132 @@
+package aws
+
+import (
+	"bytes"
+	"math/rand"
+	"testing"
+	"time"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+	"github.com/stretchr/testify/assert"
+)
+
+var uniqueTestID = "aws-nuke-test-" + uniqueID()
```

Comment:
Create this variable within each test function and not out here, as multiple test functions may run in parallel, and should each have their own unique IDs.

In fact, as a general rule, avoid any sort of global state by default!